.config/dotnet-tools.json

```diff
@@ -6,13 +6,22 @@
       "version": "7.0.0",
       "commands": [
         "fantomas"
-      ]
+      ],
+      "rollForward": false
```

Comment:
What does this do?

Reply:
i think was added by default when i installed the new command via `dotnet tool install` not sure why this is the default (i guess)?